hack/netpol/pkg/main/main.go

```diff
@@ -150,19 +150,19 @@ func main() {
 	failOnError(err)
 
 	testList := []*TestCase{
-		&TestCase{"DefaultDeny", testDefaultDeny()},
```

Comment:
I understand they are redundant but better to not touch unrelated code so that reviewers can focus on your change and there won't be surprise when seeing git history that some lines are changed in an unrelated PR.

Reply:
Removed since this is fixed in other PR.